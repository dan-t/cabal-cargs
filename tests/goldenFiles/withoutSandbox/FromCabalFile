-ilib -iexe -itests -W -optP-DLIB -optP-DEXE  -idist/build/autogen -Idist/build/autogen -optP-include -optPcabal_macros.h