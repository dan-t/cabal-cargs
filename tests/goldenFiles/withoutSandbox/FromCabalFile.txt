-ilib -iexe -itests -W -optP-DLIB -optP-DEXE -package=base -package=cmdargs -package=lens -package=directory -package=strict -package=transformers -package=either -package=text -package=system-filepath -package=system-fileio -package=Cabal -package=cabal-cargs -package=tasty -package=tasty-golden -package=filepath -idist/build/autogen -Idist/build/autogen -optP-include -optPcabal_macros.h
